kvbc/include/kvbc_app_filter/kvbc_key_types.h

```diff
@@ -36,6 +36,7 @@ const char kKvbKeySummarizedTime = 0x22;
 const char kKvbKeyCorrelationId = 0x23;
 const char kKvbKeyLastAgreedPrunableBlockId = 0x24;
 const char kIndexKey = 0x25;
+const char kKvbKeyDeleteFilesInRange = 0x26;
```

Comment:
what is the usage of this key?

Reply:
We persist the type of deletion (e.g. `delete_files_in_range`), in case of crash and restart.